README.md

```diff
@@ -38,53 +34,44 @@ overrides them in the configuration of a particular tunnel). See [example](#sett
 
 ### name
 
-By default, the role will generate a descriptive name for each tunnel it creates. For example, when creating a tunnel between  
-`bastion1` and `bastion2`, the descriptive name of this connection on `bastion1` will be `bastion1-to-bastion2`. Similarly,  
-the name of the connection associated with this tunnel on `bastion2` will be `bastion2-to-bastion1`. The user may choose to  
-override the option to auto-generate these descriptive names (which may be sizeable if hosts are identified with long FQDNs)  
-by specifying a name for the connection in this `name` field.
+By default, the role will generate a descriptive name for each tunnel it creates. For example, when creating a tunnel between `bastion1` and `bastion2`, the descriptive name of this connection on `bastion1` will be `bastion1-to-bastion2`.  
+Similarly, the name of the connection associated with this tunnel on `bastion2` will be `bastion2-to-bastion1`. The user may choose to override the option to auto-generate these descriptive names (which may be sizeable if hosts are identified with long FQDNs) by specifying a name for the connection in this `name` field.
 
 ### hosts
-Each key in this dictionary is the unique name of a host. For each host key in this dictionary, the following host-specific  
-parameters can be specified.  
+Each key in this dictionary is the unique name of a host. For each host key in this dictionary, the following host-specific parameters can be specified.  
  
 | Parameter            | Description                                                                                       | Type        | Required | Default                 | Libreswan Equivalent |
 |----------------------|---------------------------------------------------------------------------------------------------|:-----------:|:--------:|-------------------------|:--------------------:|
-| vpn_host             | An host/IP to use for setting up a vpn connection. <br /> Only specified when overriding the name of hosts used by Ansible for SSH. | str | no   | - | left/right           |
+| vpn_host             | An host/IP to use for setting up a vpn connection. Only specified when overriding the name of hosts used by Ansible for SSH. | str | no   | - | left/right           |
 | external             | Indicates that the associated host is external to the inventory (e.g. in a remote datacenter).    | bool        | no       | False                   | -                    |
 | subnet               | Defines a list of subnets of this host's public network interface.                                | list        | no       | -                       | leftsubnets/rightsubnets |
 | rsasigkey            | Defines the public key used by this host for RSA signature authentication.                        | str         | no       | -                       | leftrsasigkey/rightrsasigkey |
```

Comment:
May have to have a different name here.  https://linux-system-roles.github.io/documentation/tls_crypto_parameter_and_key_names.html

I think we should be explicit with this and `cert` below:
`shared` == PSK
`shared_key` - path to file on managed node containing a PSK
`shared_key_src` - path to a file on the controller host to be copied to the managed host to the path specified by `shared_key`
`shared_key_content` - the actual PSK in a vault secret or base64 encoded string - could be used directly in the managed host configuration e.g. in ipsec.secret - or could be the value to be copied to `shared_key`

Same with `public`
`public` == raw public key
then have `public_key,  `public_key_src`, `public_key_content`, etc.

For `cert` - this will be a little tricky - most other software we use deals with certs/keys using files in X.509 PEM encoding, and uses OpenSSL or GnuTLS which can use these files directly.  However, for libreswan, and our initial MVP, we probably don't want to have to support all of these `cert_src`, `x509_key`, etc.  I think we should use `cert_name` which differentiates this case, should be sufficient for the initial release to support certs which are already on the system, and should allow us to support other use cases in the future (e.g. if we want to support the ability of users to provide cert/key/pkcs12 files and have the role install them in the NSS cert/key database).